ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ComplianceLevelsByStandard.test.tsx

```diff
@@ -0,0 +1,162 @@
+                            numPassing: 0,
+                            numSkipped: 40,
+                            unit: 'CONTROL',
+                        },
+                        {
+                            aggregationKeys: [{ id: 'HIPAA_164', scope: 'STANDARD' }],
+                            numFailing: 8,
+                            numPassing: 10,
+                            numSkipped: 0,
+                            unit: 'CONTROL',
+                        },
+                        {
+                            aggregationKeys: [{ id: 'NIST_800_190', scope: 'STANDARD' }],
+                            numFailing: 9,
+                            numPassing: 4,
+                            numSkipped: 0,
+                            unit: 'CONTROL',
+                        },
+                        {
+                            aggregationKeys: [{ id: 'NIST_SP_800_53_Rev_4', scope: 'STANDARD' }],
+                            numFailing: 10,
+                            numPassing: 10,
+                            numSkipped: 2,
+                            unit: 'CONTROL',
+                        },
+                        {
+                            aggregationKeys: [{ id: 'PCI_DSS_3_2', scope: 'STANDARD' }],
+                            numFailing: 15,
+                            numPassing: 8,
+                            numSkipped: 1,
+                            unit: 'CONTROL',
+                        },
+                    ],
+                },
+                complianceStandards: [
+                    { id: 'CIS_Docker_v1_2_0', name: 'CIS Docker v1.2.0' },
+                    { id: 'CIS_Kubernetes_v1_5', name: 'CIS Kubernetes v1.5' },
+                    { id: 'HIPAA_164', name: 'HIPAA 164' },
+                    { id: 'NIST_800_190', name: 'NIST SP 800-190' },
+                    { id: 'NIST_SP_800_53_Rev_4', name: 'NIST SP 800-53' },
+                    { id: 'PCI_DSS_3_2', name: 'PCI DSS 3.2.1' },
+                ],
+            },
+        },
+    },
+];
+
+jest.mock('hooks/useResizeObserver', () => ({
+    __esModule: true,
+    default: jest.fn().mockImplementation(jest.fn),
+}));
+
+beforeEach(() => {
+    jest.resetModules();
+});
+
+const setup = () => {
+    const user = userEvent.setup();
+    const utils = renderWithRouter(
+        <MockedProvider mocks={mocks} addTypename={false}>
+            <ComplianceLevelsByStandard />
+        </MockedProvider>
+    );
+
+    return { user, utils };
+};
+
+describe('Compliance levels by standard dashboard widget', () => {
+    it('should render graph bars correctly order by compliance percentage', async () => {
+        const { user } = setup();
+
+        // Allow graph to load
+        await screen.findAllByRole('presentation');
+
+        async function getBarPercentages() {
+            // eslint-disable-next-line testing-library/no-node-access
+            const svgBarsElement = document.querySelector('svg > g:nth-of-type(3)');
```

Comment:
Somewhat ugly direct element access, but the chart SVG does not provide a lot of options to get the correct elements with semantic testing-library queries.